app/assets/stylesheets/custom.css.scss

```diff
@@ -208,28 +208,32 @@ img {
   min-width: 100%;
 
   h1 {
-    font-size: 6em;
     color: $share-yellow;
-    text-shadow: 4px 2px #033E58;
+    font-size: 6em;
     padding: 2em 2em 0;
     text-align: center;
+    text-shadow: 4px 2px #033E58;
```

Comment:
Color literals like `#033E58` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#033E58` should be written as `#033e58`